sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataFetcher.java

```diff
@@ -96,7 +96,7 @@ public void prepareQuickPulseDataForSend(String redirectedEndpoint) {
             request.setBody(buildPostEntity(counters));
 
             if (!sendQueue.offer(request)) {
-                logger.verbose("Quick Pulse send queue is full");
+                logger.error("Quick Pulse send queue is full");
```

Comment:
I think this is a good change after we identify the current reason for the failures (in case it's a more common reason than we think and many customers start seeing this error message and opening tickets)